docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+# Contributing to "project"
+
+The main purpose of this project is "objective"
```

Comment:
Let's replace these with
```suggestion
# Contributing to Lacus

The main purpose of this project is to develop a moderation and general purpose Discord bot.
```
I'm iffy about the objective atm, but I'm open to any other ideas.